USER_MANUAL.md

```diff
@@ -55,5 +55,25 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
   2. Additionally a handover value is incremented to monitor how many times this is called.
   3. The stack count, and the handover counter are then checked they are the same.
 
+#### QA Workflows Testing
+
+##### ReplicatedStartupActors gym
+* For QA workflows Test Replicated startup actors are correctly spawned on all clients 
+* Validation
+  1. New Play Session is started 
+  2. After two seconds checks that actor is visible to client and reports pass or fail
+
+##### DestroyingActorsAtRuntime gym
```

Comment:
If this is specifically startup actors maybe `DestroyStartupActorGym`?

Reply:
In that case it can just destroy a startup actor and then connecting another client via any means can test that it's replicated correctly. But it keeps what the gym does simple and if you want to do stuff with clients you can. Otherwise I think we are venturing into what should be tested via other means.